00-open-a-shapefile.Rmd

```diff
@@ -160,9 +163,9 @@ aoiBoundary
 Here we see that R interprets the data as a SpatialPolygonsDataFrame data class
 and information about the coordinate reference system including that the 
 coordinates  are UTMs from zone 18.  This helps us interpret the previous line 
-giving the extent of the file.  We will discuss CRS and extent a little later.
-Finally we see there is one variable named "id" that appears to consists of only 
-ones (min and max value both 1).
+giving the extent of the file.  We will discuss metadata and attributes in more
+detail a bit later. Finally we see there is one variable named "id" that appears
+to consists of only ones (min and max value both 1).
```

Comment:
This is a general question comment as I haven't dealt with this data before.  Here when just inputting object name the min and max is 1.  However, in the next section objectname@data we see that it is actually 0s and 1s.  Why the difference?  Currently the lesson points out the difference but doesn't give a reason why. Should it be added?